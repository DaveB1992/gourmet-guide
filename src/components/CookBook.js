import React from "react";
import axios from "axios";
import "../styles/cookbook.css";

const CookBook = () => {
  return (
    <ul>
      <li>recipe 1 recipe 2 recipe 3</li>
    </ul>
  );
};

export default CookBook;
