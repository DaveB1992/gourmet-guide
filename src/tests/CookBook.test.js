import React from "react";
import { render } from "@testing-library/react";
import CookBook from "../components/CookBook";

describe("CookBook", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<CookBook />);

    expect(asFragment()).toMatchSnapshot();
  });
});
