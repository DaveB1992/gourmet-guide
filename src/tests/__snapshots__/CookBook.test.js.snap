// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookBook renders correctly 1`] = `
<DocumentFragment>
  <div
    class="cookbook"
  >
    <div
      class="list"
    >
      <h2>
        My Recipes
      </h2>
      <ul
        class="vertical-list"
      />
    </div>
    <div
      class="ingredients"
    >
      <div
        class="ingredients-list"
      >
        <h2>
          Ingredients
        </h2>
        <ul
          class="vertical-list"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
